creator-node/src/routes/users.js

```diff
@@ -170,4 +170,24 @@ module.exports = function (app) {
 
     return successResponse({ clockValue })
   }))
+  /**
+   * Returns latest clock value stored in CNodeUsers entry given wallet, or -1 if no entry found
+   */
+  app.get('/users/batch_clock_status', handleResponse(async (req, res) => {
+    // TODO: Limit max size
```

Comment:
Limit max size?

Reply:
validate if this is really necessary against prod data...somehow?

Reply:
fwiw against prod data with modulo distribution, this is not actually an issue. going to table unless we end up requiring this.

Reply:
non-blocking, potentially fast follow